dbcon/ddlpackageproc/createtableprocessor.cpp

```diff
@@ -267,12 +267,17 @@ CreateTableProcessor::DDLResult CreateTableProcessor::processPackage(
         numDictCols++;
     }
 
-    fStartingColOID = fObjectIDManager.allocOIDs(numColumns + numDictCols +
-                                                 1);  // include column, oids,dictionary oids and tableoid
+    // Include column oids, dictionary oids, tableoid, and
+    // also include AUX oid as of MCOL-5021
+    fStartingColOID = fObjectIDManager.allocOIDs(numColumns + numDictCols + 2);
 #ifdef IDB_DDL_DEBUG
     cout << fTxnid.id << " Create table allocOIDs got the starting oid " << fStartingColOID << endl;
 #endif
 
+    uint32_t size = numColumns + numDictCols;
+    idbassert(size > 0);
```

Comment:
You think it is really possible?

Reply:
Not possible, I have removed this assert, as well as renamed `size` to something more meaningful.